plugins/inputs/postgresql/README.md

```diff
@@ -62,6 +62,40 @@ host=localhost user=pgotest dbname=app_production sslmode=require sslkey=/etc/te
 
 ```toml
 [[inputs.postgresql]]
-  address = "postgres://telegraf@localhost/someDB"
-  ignored_databases = ["template0", "template1"]
+  ## specify address via a url matching:
+  ##   postgres://[pqgotest[:password]]@localhost[/dbname]\
+  ##       ?sslmode=[disable|verify-ca|verify-full]
+  ## or a simple string:
+  ##   host=localhost user=pqgotest password=... sslmode=... dbname=app_production
+  ##
+  ## All connection parameters are optional.
+  ##
+  ## Without the dbname parameter, the driver will default to a database
+  ## with the same name as the user. This dbname is just for instantiating a
+  ## connection with the server and doesn't restrict the databases we are trying
+  ## to grab metrics for.
+  ##
+  address = "host=localhost user=postgres sslmode=disable"
```

Comment:
Is this a valid DSN?

Reply:
Yes. It's an [alternative format](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING) to the connection URI.

Reply:
Ok. Thanks for enlighten me! ;-)